src/html/contrast.html

```diff
@@ -1,17 +1,17 @@
 <p class="gpiic-fd-instructions gpii-fd-instructions">You can choose a screen colour to make things easier to see.</p>
 <div class="gpii-fd-controls small-offset-2 medium-offset-1 small-7 medium-9 end text-center">
     <div class="gpii-fd-choice">
-        <input class="flc-prefsEditor-theme gpii-fd-hidden-accessible" name="theme" id="theme-default" value="default" type="radio" checked>
+        <input class="flc-prefsEditor-theme gpii-fd-hidden-accessible" name="theme" id="theme-default" value="default" type="radio" tabindex="1" checked>
```

Comment:
These controls don't yet have focus styling applied to them. It makes the experience very bad. Can you look at what Jon's done for the other controls and apply it here and on the contrast page?